### PR TITLE
adds keyboard trap fix

### PR DESCRIPTION
**Pull request by esotrope**
image

image

### CODE REVIEW

File: webapp/src/main/webapp/src/app/admin/ingest-pipeline/component/pipeline-item/pipeline-item.component.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <a
   #element
+  href="javascript:void(0)"
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/admin/ingest-pipeline/component/pipeline-item/pipeline-item.component.html]: the "files" on the lhs are now tab accessible.

**Comment by malaffoon** [comment]
So, if i'm editing and i hit Escape, where am i? Sorry, not clear to me.

**Comment by esotrope** [comment]
@malaffoon 

image

**Comment by malaffoon** [comment]
So then i can hit tab to go ... ? Just want to make sure the keyboard-only navigation works reasonably well.

**Comment by esotrope** [comment]
@malaffoon tab will then go to the next form field or the browser address bar if it is the last thing on the page

File: webapp/src/main/webapp/src/app/aggregate-report/results/aggregate-report-table.component.ts [hunk 2]
```diff
@@ -127,7 +127,7 @@ function createOrderingByColumnField(
     claimCode:
       reportType === 'Target'
         ? createOrganizationalClaimOrdering(subjectDefinition.subject, preview)
-        : ordering(ranking(subjectDefinition.scorableClaims)).on(
+        : ordering(ranking(subjectDefinition.claimScore.codes)).on(
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/aggregate-report/results/aggregate-report-table.component.ts]: unrelated moving away from deprecated fields

File: webapp/src/main/webapp/src/styles.less [hunk 3]
```diff
@@ -2194,91 +2194,103 @@ fieldset.marginLeft50 {
 
 .rdw-scrollable-table {
   table,
-  tr td {
+  table > tbody > tr > td {
```
Comment (esotrope) [inline webapp/src/main/webapp/src/styles.less]: aggregate report tables are now only is one `<table>` element and have the same scrolling behavior

File: webapp/src/main/webapp/src/styles.less [hunk 4]
```diff
@@ -2194,91 +2194,103 @@ fieldset.marginLeft50 {
...
-        text-overflow: ellipsis;
-        text-align: left;
-        overflow: hidden;
+  table {
+    > thead {
+      > tr {
+        > th {
+          white-space: normal !important;
+          &.aggregate-report-organization {
+            width: @agg-organization-width;
```
Comment (esotrope) [inline webapp/src/main/webapp/src/styles.less]: we already fixed the width of all the columns so we didn't need the javascript solution that the lib gave us.

**Pull request merged**